src/Provider/RasterProvider.js

```diff
@@ -31,7 +31,11 @@ function createTextureFromVector(tile, layer) {
     if (layer.type == 'color') {
         const coords = tile.extent;
         const result = { pitch: new THREE.Vector4(0, 0, 1, 1) };
-        result.texture = Feature2Texture.createTextureFromFeature(layer.feature, tile.extent, 256, layer.style);
+        result.texture = feature2Texture(layer.feature, {
```

Comment:
so following the idea above, RasterProvider would set `layer.outputFormat = 'feature'` and use layer.transform here. And now I'm wondering if `layer.transform` shouldn't be called in the Scheduler, on line 54:

```js
return provider.executeCommand(cmd).then((result) => {
    if (cmd.layer.transform) {
        cmd.layer.transform(result);
    }
    this.counters.executing--;
    cmd.resolve(result);
    // only count successul commands
    this.counters.executed++;
}, (err) => {
    this.counters.executing--;
    cmd.reject(err);
    this.counters.failed++;
    if (__DEBUG__ && this.counters.failed < 3) {
        console.error(err);
    }
});
```

Reply:
> layer.transform shouldn't be called in the Scheduler

It should, but not in this PR. We can have the same principle as the current providers and the future parsers.